modules/weko-workflow/weko_workflow/utils.py

```diff
@@ -1469,3 +1471,22 @@ def create_files_url(root_url, record_id, filename):
         root_url,
         record_id,
         filename)
+
+
+def save_indexs_publish_doi(item_id):
```

Comment:
Can we change the method name to `update_indexes_public_state` in order to make more sense?